test/setup.js

```diff
@@ -21,6 +21,7 @@ const historicDataTo = subDays(endOfYesterday(), 200)
 const historicDataFrom = startOfDay(subDays(historicDataTo, 205))
 
 const ethAddress = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
+const source = 'ALL'
```

Comment:
`source` is used only in one test and there is no point to be shared because it's test specific.

Reply:
Alright, I will remove it then